.github/workflows/test.yml

```diff
@@ -22,10 +22,11 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Install dependencies
+      - name: Install dependencies and package
         run: |
           python -m pip install --upgrade pip
-          pip install flake8 codecov
+          pip install -r requirements.txt
+          pip install django~=${{ matrix.django-version }}
```

Comment:
It seems that this only works for major versions:
- OK: https://github.com/anexia-it/anexia-monitoring-django/actions/runs/6588250164/job/17900110329#step:4:198
- Not OK: https://github.com/anexia-it/anexia-monitoring-django/actions/runs/6588250164/job/17900110902#step:4:187